openpower/patches/p8dtu-patches/hostboot-p8/0003-VDDR-patch-for-p8dtu.patch

```diff
@@ -0,0 +1,49 @@
+From 783332cc9afcf3c8581760112162b650a9e01c7e Mon Sep 17 00:00:00 2001
+From: Jim Yuan <jim.yuan@supermicro.com>
+Date: Tue, 4 Oct 2016 16:07:44 -0700
+Subject: [PATCH 03/14] VDDR patch for p8dtu.
+
+Signed-off-by: Jim Yuan <jim.yuan@supermicro.com>
+---
```

Comment:
FYI - Happy to take a patch submission that creates a new p8dtu_vddr.C for this new logic and then puts the common code into a new file.

Reply:
A pull request is created.
https://github.com/open-power/hostboot/pull/166